lxd/auth/oidc/oidc.go

```diff
@@ -153,6 +153,13 @@ func (o *Verifier) authenticateAccessToken(ctx context.Context, accessToken stri
 		return nil, AuthError{Err: fmt.Errorf("Provided OIDC token doesn't allow the configured audience")}
 	}
 
+	// If we already have all the required information, return now.
+	res, err := o.getResultFromClaims(claims, claims.Claims)
+	if err == nil {
+		return res, nil
+	}
+
```

Comment:
in case this `err != nil` the error get swallowed. It could be useful to log it (even at debug)

Reply:
I've dropped this commit as it is not necessary for the purpose of this PR.